packages/core/src/utilities/convertStackToVolumeViewport.ts

```diff
@@ -43,7 +46,11 @@ async function convertStackToVolumeViewport({
   const { id, element } = viewport;
   const viewportId = options.viewportId || id;
 
-  const imageIds = viewport.getImageIds();
+  let imageIds = viewport.getImageIds();
+  imageIds = imageIds.map((imageId) => {
+    const imageURI = utilities.imageIdToURI(imageId);
+    return `${VOLUME_LOADER_SCHEME}:${imageURI}`;
+  });
```

Comment:
I found what's missing or the problem and tested it and it works fine. But what do you think?